internal/provisioner/helm_utils.go

```diff
@@ -198,6 +209,29 @@ func upgradeHelmChart(chart helmDeployment, configPath string, logger log.FieldL
 	return nil
 }
 
+// deleteHelmChart is used to delete Helm charts.
+func deleteHelmChart(chart helmDeployment, configPath string, logger log.FieldLogger) error {
+	arguments := []string{
+		"--debug",
+		"del",
```

Comment:
can we use the full command name? `delete`

Reply:
Done thanks :)